pyproject.toml

```diff
@@ -5,6 +5,24 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "crypt_r"
 version = "3.12.3"
+maintainers = [
+    {name = "Miro Hrončok", email = "miro@hroncok.cz"},
+    {name = "Petr Viktorin", email = "encukou@gmail.com"},
+    {name = "Tomáš Hrnčiar", email = "tomas.hrnciar@me.com"},
+    {name = "Karolina Surma", email = "ksurma@redhat.com"},
+]
+description = "A copy of the `crypt` module that was removed in Python 3.13"
+readme = "README.rst"
+# requires-python = ">=?.??" TODO
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: Python Software Foundation License",
+    "Operating System :: OS Independent",
```

Comment:
This is Linux-only. Technically it could support other Unix-like systems, but we decided to always assume crypt_r is available -- and we can only support Linux.